.dockerignore

```diff
@@ -1,5 +1,19 @@
+# python & django
+**/__pycache__/
+*.py[cod]
+*$py.class
+postgres/
+migrations/*
+!migrations/__init__.py
```

Comment:
to have docker build start with a fresh make migrations to avoid collisions with older models